recipe/meta.yaml

```
@@ -31,25 +31,25 @@ outputs:
       run_constrained:
         - {{ name }} {{ version }}
       host:
-        - python
+        - python >=3.10
         - pip
         - setuptools
         - wheel
         - packaging
-        - param >=1.9.2
-        - pyct 0.5.0
-        - bokeh ==3.4
-        - nodejs 18.16
+        - bokeh =3.5
+        - nodejs >=20
+        - hatchling
+        - hatch-vcs
       run:
-        - python
-        - bokeh >=3.4.0,<3.5.0
+        - python >=3.10
+        - bokeh >=3.5.0,<3.6.0
         - cartopy >=0.18.0
         - holoviews >=1.16.0
         - numpy >=1.0
-        - param >=1.9.0,<3.0
+        - packaging
         - panel >=1.0.0
+        - param >=1.9.3
```

Let's keep the upper pin, it's likely a major release of Param would break GeoViews in some ways. (And let's fix upstream, I have no idea why param has no pin there at all).

```suggestion
        - param >=1.9.3,<3.0
```

---

https://github.com/holoviz/geoviews/pull/752